morpheus/stages/input/rss_source_stage.py

```diff
@@ -37,14 +37,18 @@ class RSSSourceStage(PreallocatorMixin, SingleOutputSource):
     ----------
     c : morpheus.config.Config
         Pipeline configuration instance.
-    feed_input : str
+    feed_input : str, list[str]
         The URL or file path of the RSS feed.
     interval_secs : float, optional, default = 600
         Interval in seconds between fetching new feed items.
     stop_after: int, default = 0
         Stops ingesting after emitting `stop_after` records (rows in the dataframe). Useful for testing. Disabled if `0`
     max_retries : int, optional, default = 3
         Maximum number of retries for fetching entries on exception.
+    batch_size : int, optional, default = 128
+        Number of feed items to accumulate before creating a DataFrame.
+    expire_after : int, optional, default = 86400
+        Cached session expiration time in seconds.
```

Comment:
Lets switch to leaving the cache off by default and adding a new flag to turn it on when needed. Otherwise this option and `interval_secs` will clash.